### PR TITLE
Kubevirt: Update CDI to v1.57.0

### PR DESCRIPTION
**Pull request by andrewd-zededa**
Main motivator for this bump is to include
the readahead filter used in HTTP import.
This improves qcow import speed.

Part 3 of the breakup of: https://github.com/lf-edge/eve/pull/3838

### CODE REVIEW

File: pkg/kube/cluster-init.sh [hunk 1]
```diff
@@ -1,12 +1,12 @@
 #!/bin/sh
 #
-# Copyright (c) 2023 Zededa, Inc.
+# Copyright (c) 2024 Zededa, Inc.
```
Comment (eriknordmark) [inline pkg/kube/cluster-init.sh:3]: Makes sense to add years and not remove i.e., 2023-2024

Comment (andrewd-zededa) [reply pkg/kube/cluster-init.sh:3]: I'm now looking at this wondering why I bumped it, I'll fix it and drink some coffee.

**Review by eriknordmark — APPROVED** [review]
LGTM (but I need to squash the commits)

**Review by eriknordmark — APPROVED** [review]
Run again

**Pull request merged**